mcgill-en.csl

```diff
@@ -37,6 +37,18 @@
         <single>para</single>
         <multiple>paras</multiple>
       </term>
+      <term name="sub verbo" form="short">
+        <single/>
+        <multiple/>
+      </term>
```

Comment:
so yes, please remove this. We'll take up the discussion on either more (including article) or at least an empty locator again with the next CSL update and we should be able to make this work for you within regular CSL usage.